### PR TITLE
Update about page

### PR DESCRIPTION
**Pull request by geek-yang**
- Add FAQ content to about page
- Add folding/unfolding function to each question/answer
- Fetch question/answer from yaml/json file
- Polish the style of about page

### CODE REVIEW

**Comment by geek-yang** [comment]
This branch is ready for review. It already merges the latest changes in #37. To resolve the conflict, it is better to merge #37 to main branch first before merging this branch to main.

File: pages/about.vue [hunk 1]
```diff
@@ -1,62 +1,49 @@
...
+<script>
+export default {
+  data () {
+    return {
+      questions: {}
+    }
+  },
+  async mounted () {
+    const questions = await this.$content('items').fetch()
+    console.log(questions)
```
Comment (Peter9192) [inline pages/about.vue:40]:
```suggestion

```
Let's remove debug statements before deployment

File: pages/about.vue [hunk 2]
```diff
@@ -1,62 +1,49 @@
 <template>
   <div>
-    <div class="flex place-content-center bg-gray-100 m-4">
-      <h1 class="text-2xl p-6 align-middle">
-        Comparing Constrained Future European Climate Projections
+    <div class="flex place-content-center bg-gray-100 m-3">
+      <h1 class="text-xl p-3 align-middle">
+        FAQ
```
Comment (Peter9192) [inline pages/about.vue:5]: This looks super neat!

File: content/items.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    Method performance can be different in different areas or with different variables. One method, for instance, could bring significant improvements over unconstrained projections in northern Europe but not in the Mediterranean region.   It should be noted, however, that all the constraint methods used in the Atlas confer some improvement over unconstrained projections for summer temperature, and strongly related variables, and most provide improvements for all regions.
+  isActive: false
+
+- question: Where can I find more information?
+  answer: Please view the references section below for more on the background research behind this Atlas. For more on the EUCP project, please head to our website. This Atlas has been produced by Work Package 2 of EUCP, which is described in more detail here.
+  isActive: false
```
Comment (Peter9192) [inline content/items.yaml:23]: Could we include references questions, like

- Can I use these figures (yes, with attribution)
- I'd like to cite the Atlas in my paper, how can I do this?
- Is the Atlas data available as well?

File: content/items.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+items:
+- question: What do these maps show?
+  answer: This atlas shows several different model projections of the European climate in 2050, illustrating the effect of constraining the projections. Not all climate simulations are equally likely, however this is not routinely addressed in current climate projections. Methods of isolating the more likely projections include comparing a simulation of the past to observed climate data recorded at the time. This can suggest how well the simulation may project the future. This process is known as ‘constraint’ and can affect the results given by an ensemble of climate simulations by ruling out the poorer simulations.
```
Comment (Peter9192) [inline content/items.yaml:3]: There was a discussion in the email thread about potentially adding a figure to explain how "constraining methods" work. Is this still something we want to add?

**Review by Peter9192 — COMMENTED** [review]
Really nice work so far! I think the functionality is awesome, we just might want to add a few more questions/content

**Review by Peter9192 — APPROVED** [review]
Really nice work! I implemented some of my own suggestions. Merging now, made #47 to finalize the content.

**Pull request merged**